### ISSUE TITLE
Hamburger background hardcoded?

### ISSUE THREAD

**Issue by GokulNC** [sylque/discpage#2]
When I installed the plugin, I see that the background of my header is different from that of the hamburger's:
![image](https://user-images.githubusercontent.com/10559293/75230390-8c349f00-57d9-11ea-9df6-ef39c75874d7.png)

Upon inspecting, I see that it's coming from the `discpage` plugin:
![image](https://user-images.githubusercontent.com/10559293/75230486-b4240280-57d9-11ea-96fc-40d8e71713c2.png)

Please fix this.

**Comment by sylque** [comment]
The red background of the hamburger menu means there's an issue with DiscPage. To come back to normal, you need either:
- to fix the underlying DiscPage issue, or 
- to disable the plugin (using the `discpage enabled` setting).

The underlying issue should appear in your debug console. Can you see a DiscPage-related error rmessage there?

**Comment by GokulNC** [comment]
Yes, I see the following error:
```
DiscPage Error - Invalid Discourse setting "discpage page_categories": not set
```

For now, I have the plugin enabled, but have not added any categories for DiscPage.
So, isn't this a normal scenario? Should we really treat this as an error?

**Comment by sylque** [comment]
> Should we really treat this as an error?

You're right, there certainly is something to change here. I could either:
1. treat an empty "discpage page categories" setting as normal, or
2. have "discpage enable" false by default.

I lean toward 2, because I don't want users to miss the "discpage page categories" and then to wonder why it doesn't work. What do you think?

**Comment by GokulNC** [comment]
Maybe as a third option, you could:

3. Add a setting called `discpage show errors`. By default, it can be enabled. When there's an error, only if this boolean is enabled, you can change the hamburger color.

You can add a description for this setting indicating that if enabled, hamburger will turn red if there are errors.

**Comment by sylque** [comment]
> Maybe as a third option, you could:

I don't know. Maybe this "red hamburger menu" is not a good idea and should be removed. I'll stick to option 2 for now (as it is the easiest to implement) and wait for more feedback.

**Comment by sylque** [comment]
Closing this right now. Option 2 has been implemented in v1.0.14 and more info has been added to the doc. We'll see if future users are still puzzled.
